packages/babel-plugin-transform-es2015-unicode-regex/package.json

```diff
@@ -11,7 +11,9 @@
   "dependencies": {
     "babel-helper-regex": "^6.8.0",
     "babel-runtime": "^6.0.0",
-    "regexpu-core": "^1.0.0"
+    "babel-helper-regex": "^6.3.13",
+    "babel-runtime": "^5.0.0",
```

Comment:
I think this should be 6.0.0 since it was before, and `babel-helper-regex` is duplicated